### PR TITLE
Fix pixelated initials in avatar

### PR DESCRIPTION
**Pull request by johnnyoin**
Micro commit which fixes pixelated initials in avatars

### CODE REVIEW

**Review by SD10 — DISMISSED** [review]
Good one @johnnyoin 🥇

File: Sources/Views/AvatarView.swift [hunk 1]
```diff
@@ -89,7 +89,7 @@ open class AvatarView: UIView {
         if width == 0 || height == 0 {return UIImage()}
         var font = placeholderFont
 
-        _ = UIGraphicsBeginImageContext(CGSize(width: width, height: height))
+      _ = UIGraphicsBeginImageContextWithOptions(CGSize(width: width, height: height), false, UIScreen.main.scale)
```
Comment (Sherlouk) [inline Sources/Views/AvatarView.swift]: Doesn't look like we ever end this image context 🤔 Worth chucking it in a defer?

Comment (SD10) [reply Sources/Views/AvatarView.swift]: Yep, good point 👍

**Comment by SD10** [comment]
@johnnyoin Can you make the change discussed above? Also, let's add a CHANGELOG entry under `Fixed` for fixing the pixelated avatars?

**Review by SD10 — APPROVED** [review]

**Comment by SD10** [comment]
Thank you for contributing to MessageKit! I've invited you to join the MessageKit GitHub organization - no pressure to accept! If you'd like more information on what that means, check out our [contributing guidelines](https://github.com/MessageKit/MessageKit/blob/master/CONTRIBUTING.md) and join the [MessageKit Slack channel](https://join.slack.com/t/messagekit/shared_invite/enQtMjI4NzIzNzMyMzU0LTEyNmYzNzBlNjNiYjAwYjY1ZjhkMmVjNDE4OGNlMDY1OTRmNzVmMzI4MTg3OWQzZTM2NGU4NTU3ZDE5NzkwNzQ). Feel free to reach out if you have any questions! :smiley:

**Pull request merged**